core/bin/zksync_api/src/fee_ticker/validator/watcher.rs

```diff
@@ -14,61 +14,116 @@ pub trait TokenWatcher {
     async fn get_token_market_volume(&mut self, token: &Token) -> anyhow::Result<BigDecimal>;
 }
 
-/// Watcher for Uniswap protocol
-/// https://thegraph.com/explorer/subgraph/uniswap/uniswap-v2
+#[derive(Serialize, Deserialize, Debug)]
+struct CGTokenTotalVolume {
```

Comment:
NIT: I'd prefer to have a clear name rather than a short one (e.g.: `CoinGeckoTokenTotalVolume`)

Reply:
I changed it overall. I copied all types from coingecko api to a separate file called `types.rs`.